### PR TITLE
Do not use bindtointerface on windows (CFE-2662)

### PR DESCRIPTION
**Pull request by olehermanse**
Note that this does not fix the issue, it's just a workaround on windows.

### CODE REVIEW

**Comment by olehermanse** [comment]
@nickanderson This should be merged and cherry picked to 3.11.x

File: controls/cf_serverd.cf [hunk 1]
```diff
@@ -32,9 +32,6 @@ body server control
       # Suggested value >= total number of hosts
       maxconnections        => "$(def.control_server_maxconnections)";
 
-      # Bind to all interfaces including ipv6
```
Comment (nickanderson) [inline controls/cf_serverd.cf]: If you remove this definition from the implicit any:: context then the agent will only bind to ipv4 interfaces, so this one should remain, and then be overridden for windows below.

Comment (olehermanse) [inline controls/cf_serverd.cf]: @nickanderson Did you see the whole context? This is inside `!windows` context. So should apply just like before on everything except windows.

Comment (olehermanse) [inline controls/cf_serverd.cf]: Also, this is `body server control` so why do you mention agent?

Comment (jimis) [inline controls/cf_serverd.cf]: @olehermanse I think @nickanderson 's suggestion will have the same effect with what you did in this patch.

Comment (olehermanse) [inline controls/cf_serverd.cf]: @jimis @nickanderson I have tested with and without this patch on linux now. Even tested ipv6. Cannot see any difference, works fine with and without this patch.

**Pull request merged**